rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -245,11 +258,47 @@ void RingBuffer::initBoxEncoderMetaData(RocalMemType mem_type, size_t encoded_bb
     }
 #else    
     {
-        //todo:: for host
+        if(_meta_data_sub_buffer_count < 2)
+            THROW("Insufficient HOST metadata buffers for Box Encoder");
+        // Check if sufficient data has been allocated for the labels and bbox host buffers if not reallocate
+        for(size_t buffIdx = 0; buffIdx < BUFF_DEPTH; buffIdx++)
```

Comment:
if BUFF_DEPTH is not a constant, change it to lowercase

Reply:
It is there in TOT also hence not making this change here